certification/README.md

```diff
@@ -0,0 +1,2 @@
+# certification
+Software Carpentry certifications
```

Comment:
Since this is one kind of dependency we can't automate installing/updating by normal tools, please add a note to `docs/` to mark where it comes from.